pages/windows/scoop-bucket.md

```diff
@@ -0,0 +1,25 @@
+# scoop bucket
+
+> A bucket is a Git repository containing files which describe how scoop installs applications.
+> If Scoop doesn't know where the bucket is located we have to specify its repository location.
```

Comment:
Perhaps this will read better?
```suggestion
> Manage buckets: Git repositories containing files which describe how scoop installs applications.
> If Scoop doesn't know where the bucket is located its repository location must be specified.
```

Reply:
Yeah, that's even better